File: backend/models/matchModel.js
const mongoose=require('mongoose');
const Schema=mongoose.Schema;

const MatchSchema=new Schema({
    userId:{//maç kullanıcı ilişkisi için
        type:mongoose.Types.ObjectId,
        ref:'user',
        required:true
    },
    matchDate:{//maç tarihi
        type:Date,
        required:true
    },
    matchStart:{//maç başlangıç saati
        type:String
    },
    matchStop:{//maç bitiş saati
        type:String
    },
    stadiumId:{//maç stad ilişkisi için
        type:mongoose.Types.ObjectId,
        ref:'stad'
    },
    service:{//maç için servis durumunu belirtir
        type:Boolean,
        default:false
    },
    note:{//maça kayıtlı not bilgisi
        type:String
    },
    totalPrice:{//maça ait toplam ücret (maç ve varsa servis ücreti)
        type:Number
    }
},{collation:'match'})

module.exports=mongoose.model('match',MatchSchema)

File: HaliSahaRezervasyonPortali/HaliSahaRezervasyonPortali/Models/matchModel.cs
using Newtonsoft.Json;
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Web;

namespace HaliSahaRezervasyonPortali.Models
{
    public class matchModel
    {
        [JsonProperty("_id")]
        public string _id { get; set; }

        [JsonProperty("userId")]
        public string userId { get; set; }

        [JsonProperty("matchDate")]
        public string matchDate { get; set; }

        [JsonProperty("matchStart")]
        public string matchStart { get; set; }

        [JsonProperty("matchStop")]
        public string matchStop { get; set; }

        [JsonProperty("stadiumId")]
        public string stadiumId { get; set; }

        [JsonProperty("service")]
        public bool service { get; set; }

        [JsonProperty("note")]
        public string note { get; set; }

        [JsonProperty("totalPrice")]
        public double totalPrice { get; set; }
    }
}